Makefile

```diff
@@ -1,5 +1,12 @@
 PREFIX ?= /usr
 
-install: nix-bundle.sh nix-run.sh appdir.nix appimagetool.nix appimage.nix AppRun.c appimage-top.nix default.nix appdir.sh nix2appimage.sh
+.PHONY: install
+
+install:: nix-bundle.sh nix-run.sh appdir.nix appimagetool.nix appimage.nix AppRun.c appimage-top.nix default.nix appdir.sh nix2appimage.sh
```

Comment:
I'm confused what's going on here- but I think I've resolved this issue in dcbfa1a76189710503a664bcaf9aea206ae78beb.

Reply:
Different approaches, I wasn't sure of how this was intended to be used so tried to be more careful about modifying things :).

I'll rebase+squash as requested, and drop this.